streamalert_cli/terraform/cloudwatch.py

```diff
@@ -18,7 +18,32 @@
 LOGGER = get_logger(__name__)
 
 
-def generate_cloudwatch(cluster_name, cluster_dict, config):
+def generate_cloudwatch_destinations(cluster_name, cluster_dict, config):
+    cloudwatch_module = config['clusters'][cluster_name]['modules']['cloudwatch_destinations']
+    if not cloudwatch_module.get('enabled'):
+        LOGGER.debug('CloudWatch destinations module is not enabled')
+        return True  # not an error
+
+    account_ids = cloudwatch_module.get('cross_account_ids', [])
+    regions = cloudwatch_module.get('regions')
+    if not regions:
+        LOGGER.error(
+            'CloudWatch destinations must be enabled for at '
+            'least one region in the \'%s\' cluster',
+            cluster_name
+        )
+        return False
+
+    return _generate(cluster_name, cluster_dict, config, account_ids, regions)
+
+
+def generate_cloudwatch_destinations_internal(cluster_name, cluster_dict, config):
```

Comment:
A short comment here will be very helpful what is the different between this function and `generate_cloudwatch_destinations` function.